packages/patternfly-4/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -66,8 +66,8 @@ export class InternalDropdownItem extends React.Component<InternalDropdownItemPr
       keyHandler: () => {},
       sendRef: () => {}
     },
-    id: '',
-    componentID: '',
+    id: null,
+    componentID: null,
```

Comment:
Instead of setting this to null, couldn't we just not set a default? That is, remove this from `defaultProps`.

We shouldn't need to test `componentID || undefined` because it would already be undefined.

Reply:
that was my thought as well, but if i recall the lint settings require optional props to have defaults, though `undefined` may be a valid default 🤷‍♂

Reply:
yeah if you remove the default props here then it will just default to undefined and be fine.